ui/components/MesheryPerformance/Dashboard.js

```diff
@@ -112,7 +112,7 @@ function Dashboard({ updateProgress, enqueueSnackbar, closeSnackbar, grafana, ro
             <Grid container spacing={1}>
               <Grid item xs>
                 <Paper className={classes.paper}>
-                  <div style={{ display: "flex", alignItems: "center" }}>
+                  <div style={{ display: "flex", alignItems: "center" , height: "6.8rem"}}>

```

Comment:
why are we using fixed height here ?

Reply:
yes this is for the `service mesh metrics` card and the `profiles and results` card to be the same height

Reply:
hmm ,actually, the `service mesh metrics` will expand when users have grafana or promethues configured, so, in that case, having a fixed height will not be desired

Reply:
oops, sorry my bad, the fixed heights are for the `results` and `profiles` card and not for the `service mesh metrics` , would that still be a problem?

Reply:
Ok 👍🏽